src/main/reactjs/src/utils/index.ts

```diff
@@ -73,6 +76,27 @@ export class Utils {
     return notifier;
   }
 
+  static createNotifierToastForAxiosError(error: AxiosError) {
+    const t = translateOutsideComponent();
+    const apiError = axios.isAxiosError(error)
+      ? Utils.getAPIError(error)
+      : undefined;
+
+    const message = apiError
+      ? t(`akt.api.errors.${apiError}`)
+      : t('akt.api.errors.generic');
+
+    return Utils.createNotifierToast(Severity.Error, message);
+  }
+
+  static getAPIError(error: AxiosError): string | undefined {
+    if (error.response && error.response.data.errorCode !== undefined) {
+      const code = error.response.data.errorCode;
+
+      return Object.values(APIError)[code];
```

Comment:
Mitä jos indeksillä ei löydy, tulee virheviesti akt.api.errors.undefined?

Reply:
Pitäis kai tulla `akt.api.errors.generic` ? Vai ajattelenko tän nyt jotenkin väärin?

Reply:
Aivan, olet oikeassa.